### ISSUE TITLE
how to modify accessLog time zone

### ISSUE THREAD

**Issue by xiaowujee** [containous/traefik#4158]
Bug

###

I have modified the server time zone.The traefikLog time zone has been modified,but accessLog time zone no change.
![image](https://user-images.githubusercontent.com/26532548/47984670-45b5bf80-e112-11e8-8811-d46dbeb3af74.png)

### traefik version

 1.7.4

### docker image

docker pull traefik:1.7.4

```toml
[accessLog]
      bufferingSize = 100
      [accessLog.filters]
        statusCodes = ["100-510"] 
        [accessLog.fields]
        defaultMode = "keep"
        [accessLog.fields.headers]
          defaultMode = "keep"
```

**Comment by ldez** [comment]
Closed in favor of https://github.com/containous/traefik/issues/3870#issuecomment-440826268

**Comment by imro2** [comment]
> 
> 
> Closed in favor of [#3870 (comment)](https://github.com/containous/traefik/issues/3870#issuecomment-440826268)

The solution to that ticket specifically talks about JSON formatted log. So it is not really the same thing. According to wikipedia the Common Log Format date field contains "the date, time, and time zone that the request was received". So the limitation is on Traefik's side.
